Python/PR_Analyzer/README.md

```diff
@@ -0,0 +1,87 @@
+# Pull Request Analysis
```

Comment:
Please follow this: https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md

Reply:
Done!